### PR TITLE
Lion 116 Add algorithm to return challenge progress to Home

### PR DESCRIPTION
**Pull request by rena52107**
[Desc]
- Added algorithm to check daily challenge progress
- Return ongoing challenges data to Home screen

[Todo]
- Check event/coop challenge progress
- Make card component to display challenge content
- Make Button triggering Success Modal when challenge achieved

### CODE REVIEW

File: src/api/constants.js
Comment (nightfallb612) [inline src/api/constants.js:1]: I modified this file yesterday.
I think you include this by mistake.
Could you omit this file.

File: src/services/checkChallengeAchievement.js [hunk 1]
```diff
@@ -32,8 +17,142 @@ async function checkChallengeAchievement() {
...
+
+//   try {
+//     const result = await updateUserInfo(userInfo, updatedUserInfo);
+
+//     return returnObj;
+//   } catch (error) {
+//     console.log("Error in increaseStreakDays");
+//     throw error;
+//   }
+// };
```
Comment (nightfallb612) [inline src/services/checkChallengeAchievement.js]: This comment block is going to use later? or please delete this.

**Review by nightfallb612 — CHANGES_REQUESTED** [review]
Could you check specific comment? and request to review again.

File: src/api/constants.js
Comment (rena52107) [reply src/api/constants.js:1]: Ok, fixed!

**Review by nightfallb612 — APPROVED** [review]
Review completed.

**Pull request merged**